apps/web/src/views/Home/components/Banners/WebNotificationBanner.tsx

```diff
@@ -196,20 +197,14 @@ const WebNotificationBanner = () => {
     >
       <S.Inner>
         <S.LeftWrapper position="relative" style={{ zIndex: 2 }}>
-          <Box height={isMobile ? '20px' : '29px'} mt={isMobile ? '-10px' : '0px'}>
-            {isMobile ? (
-              <Image src={pancakeSwapLogo} alt="pancakeSwapLogo" width={124} height={16} unoptimized />
-            ) : (
-              <Image src={pancakeSwapLogo} alt="liquidStakingTitle" width={136} height={20} unoptimized />
-            )}
-          </Box>
-          <StyledSubheading data-text={t('Web3 Notifications (BETA) trial available')}>
-            {t('Web3 Notifications (BETA) trial available')}
+          <PancakeSwapBadge />
+          <StyledSubheading data-text={t('Web3 Notifications Now Live!')}>
+            {t('Web3 Notifications Now Live!')}
           </StyledSubheading>
           {!isMobile && <Header>{t('All Your DeFi Updates, All in One Place')}</Header>}
           <Flex style={{ gap: isMobile ? 4 : 16 }}>
             <Link
-              href="https://blog.pancakeswap.finance/articles/introducing-web3-notifications-on-pancake-swap"
+              href="https://blog.pancakeswap.finance/articles/pancake-swap-s-web3-notification-official-launch-is-live"
```

Comment:
Wrong URL.
should be

`https://blog.pancakeswap.finance/articles/pancake-swap-s-web3-notifications-official-launch-is-live`